sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added [`sycl_ext_intel_grf_size`](doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc) extension. [83d099758770]
+- Added proposal for [`sycl_ext_oneapi_append_and_shift`](doc/extensions/proposed/sycl_ext_oneapi_append_and_shift.asciidoc) extension. [1a283acaac3c]
+- Updated description of accessor-based memory APIs in [`sycl_ext_intel_esimd`](sycl/doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md). [9e5889918277]
+- Added draft of [`sycl_ext_oneapi_prefetch`](sycl/doc/extensions/proposed/sycl_ext_oneapi_prefetch.asciidoc). [034ac0958662]
+- Added draft of [`sycl_ext_oneapi_barrier`](doc/extensions/proposed/sycl_ext_oneapi_barrier.asciidoc) [32144edcf982]
+- Added legacy SYCL 1.2.1 image aspect. [78b709dd5bb8]
+- Added [`sycl_ext_intel_data_flow_pipes_properties`](sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc) extension. [5b0a4615874b]
+- Updated [`sycl_ext_intel_dataflow_pipes`](doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc) extension for FPGA host. [6b2d66b8d52f]
+
+## Improvements
+
+### SYCL Compiler
+- Started to use LLVM IR instead of GenX intrinsics for ESIMD block loads/stores - USM and SLM - and group/thread id/size queries. [8af94a01f063][7e7971a57666]
+- Deprecated -fsycl-link-huge-device-code in favor of a new option, -flink-huge-device-code. The new option is identical in functionality but allowed with -fopenmp-targets. [79ea9b7aa52b]
+- Clean up device_globals from llvm.compiler.used. [f99335f5e71a]
+- Started to emit an error when PCH is triggered in SYCL mode. [062446584d52]
+- Improved FPGA archive device unbundling with AOCO. [4ed3676f6402]
+- Moved imf `abs` to a separate device library for Deep Learning. [e4f074a6aabe]
+- Fixed the bug report URL for DPC++. [98b7de88a7d9]
+
+### SYCL Library
+- Combined ADL-S and RPL-S device architectures. [f87be6f17428]
+- Implemented multi_ptr default to be legacy to avoid code break with SYCL 1.2.1. [52edb3798749]
+- Started to use aggregate initialization for `group_local_memory` arrays according to updated [sycl_ext_oneapi_local_memory](sycl/doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc).[b50440ec76b9]
+- Deprecated ESIMD Emulator.  [177680e3e918]
+- Deprecated `ext::oneapi::sub_group`. [0662e2a81014]
+- Improved error message related to `ext_intel_free_memory` aspect. [c1bfcaf6d6ed]
+- Removed non-necessary barrier after region copy in the Level Zero backend. [768e61e7966c]
+- Fixed `get_info<device::free_memory>` to check its aspect. [dfb75675bc0a]
+- Removed the workaround for release of auxiliary buffers. [fda9171fc218]
+- Enabled optimization for read-only buffer accesses from multiple devices in a context. [dddaf7f5d54d]
+- Removed old specialization constants extension and implementation. [e50ae05e28d3][833a9fe0a9cf]
+- Added README.md for the Unified Runtime directory. [453ffdcf41dd]
```

Comment:
```suggestion
```

Alternatively we can move this to "Documentation" section

Reply:
Moved to Documentation.